vyper/ast/nodes.py

```diff
@@ -937,7 +937,9 @@ class Invert(Operator):
     __slots__ = ()
     _description = "bitwise not"
     _pretty = "~"
-    _op = operator.inv
+
+    def _op(self, value):
+        return (2 ** 256 - 1) - value
```

Comment:
this is only valid for uint256 btw

Reply:
it's going to be clearer imo as
```suggestion
        return (2 ** 256 - 1) ^ value
```
the xor reflects what's going on with the bits. it's also going to generalize to other integer types a bit better. @tserg what do you think?

Reply:
I agree, I started with a xor mask before changing to its current form to match what was already in `vyper.utils`.